tezos/context/src/working_tree/storage.rs

```diff
@@ -485,14 +513,19 @@ pub struct Storage {
     blobs: Vec<u8>,
     /// Concatenation of all strings in the working tree.
     /// The working tree has `StringId` which refers to a data inside `StringInterner`.
-    pub strings: StringInterner,
+    // pub strings: Option<&'a mut StringInterner>,
```

Comment:
this is commented out here, not used anymore? (iirc it was not part of the index or repository), if so lets remove it